specification/compute/resource-manager/Microsoft.Compute/preview/2020-09-30/gallery.json

```diff
@@ -191,7 +191,7 @@
               "Permissions"
             ],
             "x-ms-enum": {
-              "name": "Permissions",
+              "name": "SelectPermissions",
```

Comment:
this will be SDK breaking change

Reply:
Yes, I need to make this change for a successful SDK generation. This is in preview, and no one is using it.